django-hosting-heroku/README.md

```diff
@@ -0,0 +1,45 @@
+# Hosting a Django Project on Heroku
+
+This is a companion project to a Real Python tutorial about [Hosting a Django Project on Heroku](https://realpython.com/django-hosting-on-heroku/). It's a bare-bones Django project without any views.
+
+## Instructions
+
+You should follow the step-by-step instructions in the tutorial to deploy this project to Heroku. Alternatively, you can run the finished project by following the instructions below.
+
+Download the code by cloning the `materials` repository:
+
+```shell
+$ git clone git@github.com:realpython/materials.git
```

Comment:
Is there a reason you're not using the `https://` link of the repository, like [suggested in GitHub's docs](https://github.com/git-guides/git-clone)?

```suggestion
$ git clone https://github.com/realpython/materials.git
```

Reply:
It's muscle memory :slightly_smiling_face: I'm so used to cloning with SSH to avoid typing the password manually that I forgot the materials repository is public. By the way, the resource you linked to doesn't seem to recommend one over the other method as far as I can tell.